templates/01-hello-username/setup.js

```diff
@@ -4,8 +4,8 @@
 require.config({
   paths: {
     "binaryen": "https://cdn.jsdelivr.net/gh/AssemblyScript/binaryen.js@84.0.0-nightly.20190522/index",
-    "assemblyscript": "https://cdn.jsdelivr.net/gh/nearprotocol/assemblyscript@d8c4f1de18614c966f19b41f89ace2ef5cd876c5/dist/assemblyscript",
-    "assemblyscript/bin/asc": "https://cdn.jsdelivr.net/gh/nearprotocol/assemblyscript@d8c4f1de18614c966f19b41f89ace2ef5cd876c5/dist/asc"
+    "assemblyscript": "https://raw.githack.com/nearprotocol/assemblyscript/package_management/dist/assemblyscript",
+    "assemblyscript/dist/asc": "https://raw.githack.com/nearprotocol/assemblyscript/package_management/dist/asc"
```

Comment:
any reason to use `githack.com` instead of `jsdeliver.net`? Also I think we'd better pin specific version of compiler, as using latest in branch is error prone (existing studio projects might get broken more quickly) and can have issues with caching.

Reply:
Yeah this was mainly for testing purposes since I want changing the compiler to fix bugs as they came up as this was the first time I could test it in a browser setting.